components/next_steps_view/next_steps_view.tsx

```diff
@@ -183,8 +184,15 @@ export default class NextStepsView extends React.PureComponent<Props, State> {
         );
     }
 
+    // Filter the steps shown by checking if our user has any of the required roles for that step
+    filterSteps = (step: StepType) => {
+        return isStepForUser(step, this.props.currentUser.roles);
+    }
+
+    steps = Steps.filter(this.filterSteps);
```

Comment:
This is the sort of thing that should be handled by a redux selector. We should try and avoid adding global constants to a class like this.

Reply:
Good callout. This is my redux noobiness showing. Much cleaner with the selector.